posthog/src/main/java/com/posthog/internal/GzipRequestInterceptor.kt

```diff
@@ -52,7 +52,7 @@ internal class GzipRequestInterceptor(private val config: PostHogConfig) : Inter
                 try {
                     originalRequest.newBuilder()
                         .header("Content-Encoding", "gzip")
-                        .method(originalRequest.method, forceContentLength(gzip(body)))
+//                        .method(originalRequest.method, forceContentLength(gzip(body)))
```

Comment:
Android errors with gzip using flags but not with /decide
[posthog.slack.com/archives/C07Q2U4BH4L/p1749803054185859](https://posthog.slack.com/archives/C07Q2U4BH4L/p1749803054185859)